### PR TITLE
EPMRPP-61094: [AGENT] Postman. ECONNREFUSED with reporter only when using CLI

### PR DESCRIPTION
**Pull request by KatsiarynaTatarynovich**

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -124,12 +124,12 @@ module.exports = {
         };
 
         return {
-            launch: options.launch || DEFAULT_LAUNCH_NAME,
-            description: options.description,
+            launch: options.launch || options.reportportalAgentJsPostmanLaunch || DEFAULT_LAUNCH_NAME,
+            description: options.description || options.reportportalAgentJsPostmanDescription,
             attributes: options.attributes ? options.attributes.concat(systemAttr) : [systemAttr],
```
Comment (AmsterGet) [inline lib/utils.js:129]: Don't we need to use `options.reportportalAgentJsPostmanAttributes` here?

**Review by AmsterGet — CHANGES_REQUESTED** [review]
Also please add a note about this fix to the `CHANGELOG.md` with link to the issue.

File: lib/utils.js (hunk 1 above)
Comment (KatsiarynaTatarynovich) [reply lib/utils.js:129]: The attributes will be implemented as part of the other task [EPMRPP-61095](https://jiraeu.epam.com/browse/EPMRPP-61095)

**Comment by KatsiarynaTatarynovich** [comment]
> Also please add a note about this fix to the `CHANGELOG.md` with link to the issue.

I'm going to add the note in the final PR before release. Are you ok with it?

File: lib/utils.js (hunk 1 above)
Comment (AmsterGet) [reply lib/utils.js:129]: Got it.

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**